src/CreatorToken.sol

```diff
@@ -179,10 +249,19 @@ contract CreatorToken is ERC721Royalty {
     payToken.safeTransfer(admin, _adminFee);
   }
 
+  /// @notice Sell multiple tokens in bulk.
+  /// @dev Sets the minimum accepted proceeds to 0.
+  /// @param _tokenIds Array of token IDs to be sold.
+  /// @return _netProceeds The total net proceeds in USDC from the bulk sale.
   function bulkSell(uint256[] memory _tokenIds) public returns (uint256 _netProceeds) {
     _netProceeds = bulkSell(_tokenIds, 0);
   }
 
+  /// @notice Sell multiple tokens with a minimum accepted proceeds in USDC for the total sale.
+  /// @param _tokenIds Array of token IDs to be sold.
+  /// @param _minAcceptedPrice The minimum total proceed in USDC the seller is willing to accept for
```

Comment:
nit: should say "total *net* proceeds"

Reply:
changed in https://github.com/showtime-xyz/creator-tokens/commit/3c46721e1c9a0899f54630e2b73028cd5bef159c